<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMain</class>
 <widget class="QWidget" name="WidgetMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leap Year</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="appresources.qrc">:/img/infotronicaH.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Specify the year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Find Out</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Year in binary format</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="labelResult">
       <property name="text">
        <string>Im Ready</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Verify February days</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="gridVisible">
        <bool>true</bool>
       </property>
       <property name="dateEditEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Brief Explanation&lt;/p&gt;&lt;p&gt;The first two binary digits says LEAP or NO LEAP year&lt;/p&gt;&lt;p&gt;using the &amp;amp; binary operator&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;XXXXXXXX&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#ff0000;&quot;&gt;00&lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt; &amp;amp; &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#000080;&quot;&gt;3 &lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;= &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#000080;&quot;&gt;0 --&amp;gt; &lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;LEAP YEAR&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;XXXXXXXX&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#ff0000;&quot;&gt;01&lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt; &amp;amp; &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#000080;&quot;&gt;3 &lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;= 1 --&amp;gt; NO LEAP&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;XXXXXXXX&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#ff0000;&quot;&gt;10&lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt; &amp;amp; &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#000080;&quot;&gt;3 &lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;= 1 --&amp;gt; NO LEAP&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;XXXXXXXX&lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#ff0000;&quot;&gt;11&lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt; &amp;amp; &lt;/span&gt;&lt;span style=&quot; font-family:'monospace'; color:#000080;&quot;&gt;3 &lt;/span&gt;&lt;span style=&quot; font-family:'monospace';&quot;&gt;= 1 --&amp;gt; NO LEAP&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Finally we apply the ! operator for true when is LEAP YEAR &lt;/p&gt;&lt;p&gt;and false when is not LEAP YEAR&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="appresources.qrc"/>
 </resources>
 <connections/>
</ui>
